src/Schema/index.ts

```diff
@@ -1,5 +1,6 @@
 export * from "./Events"
 export * from "./Events/Authentication"
+export * from "./Events/Conversations"
 export * from "./Events/Click"
```

Comment:
@eessex A organization questions for events, looks like we have both actions (e.g. Click, Tap, etc.) and functionalities (e.g. Authentication, Conversations, etc.). Curious about guidelines for organizing new events.

Reply:
I think conversations makes sense if we expect more events to exist in this domain.